### ISSUE TITLE
Does not classify programming files

### ISSUE THREAD

**Issue by pdoongarwal** [bhrigu123/classifier#49]

**Comment by leokhachatorians** [comment]
Should be an easy fix, just need to add a bunch of common extensions.

**Comment by pdoongarwal** [comment]
@bhrigu123 Can I fix this? Or someone is already doing that?

**Comment by bhrigu123** [comment]
Hi, I was wondering if a user would really need to cluster all his different codes into a single folder. I mean he wouldn't like to take all the `.cpp`, `.java`, `.py`...etc, files and keep them in a single folder.
In fact it would be better to create different folders ("Java FIles", "Python Codes") for different types of codes.
And for that he could use the `-st` option.

**Comment by pdoongarwal** [comment]
@bhrigu123 we can make sub folders in a main folder according to languages.
I myself code in many languages, sometimes it gets difficult to maintain files, it would be really helpful if you can add this feature.
You can't expect users to use ``` -st``` command multiple(6-8 different languages) times.

**Comment by leokhachatorians** [comment]
Yeah that's true. I think if auto completion is a focus then having features such as this are ideal.

**Comment by pdoongarwal** [comment]
@bhrigu123 should I send a PR!?

**Comment by bhrigu123** [comment]
I'll let you know. I would like to see if more people are up for this for now.
( however for your own use, you can edit the downloaded package and install it using `setup.py` if you want, for now )

**Comment by pdoongarwal** [comment]
@bhrigu123 Ok

**Comment by juliusdanek** [comment]
+ 1
